chaco/scales/tests/test_time_scale.py

```diff
@@ -1,61 +1,240 @@
 
 
-import time
+import datetime
+import os
+import contextlib
 
 import numpy as np
 
-from chaco.scales.time_scale import tfrac, trange, \
-        TimeScale, CalendarScaleSystem
+from chaco.scales.time_scale import (
+    tfrac, trange, TimeScale, CalendarScaleSystem)
 from chaco.scales.api import TimeFormatter
 
+# Note on testing:
+# Chaco assumes times are in UTC seconds since Posix Epoch but does ticking
+# in whatever the local time is on the host machine.  This is problematic for
+# testing, because correct responses depend on current state of the test
+# machine, and will vary by location and time of year.  For hour/day/year
+# testing, where TZ matter, we select 3 timezones to test in:
+#
+# * UTC, because it's easy to reason about what the answer should be
+# * Australia/North (ie. Alice Springs, Darwin) which is UTC+9:30 and with no
+#   daylight savings time shifts, which should hopefully reveal bugs with
+#   non-integer hour offsets.
+# * Pacific/Honolulu which is UTC-10:00 and with no daylight savings time
+#   shifts, which should hopefully reveal bugs with previous-day or -year
+#   baselines.
+#
+# Note that this isn't quite bullet-proof, as things will break if Hawaii or
```

Comment:
Hmmm... that will probably work.  I was concerned that there may be other calls to the underlying datetime code that rely on system time.

On the other hand, when it is working on OS X and Travis CI as intended (even if it's not 100% portable or reliable) that makes me somewhat happier about the robustness of the code, since it protects against future errors introduced if people use something which relies on `datetime` incorrectly.

So maybe the right fix is to skip these tests on windows?